src/Mount.cpp

```diff
@@ -1402,7 +1397,10 @@ void Mount::startSlewingToHome()
 /////////////////////////////////
 void Mount::stopGuiding()
 {
-    stopGuiding(true, true);
+    if (isGuiding())
```

Comment:
The conditional should be in `stopGuiding(bool ra, bool dec)`, not `stopGuiding()`